src/components/SoundWave.jsx

```
@@ -0,0 +1,70 @@
+  };
+
+  return (
+    <div>
+      <svg
+        width="302"
+        height="28"
+        viewBox="0 0 344 28"
+        fill="none"
+        xmlns="http://www.w3.org/2000/svg"
+        style={{ maxWidth: "100%", height: "auto" }}
+      >
```

Biome (2.1.2)

[error] Alternative text title element cannot be empty

For accessibility purposes, SVGs should have an alternative text, provided via title element. If the svg element has role="img", you should add the aria-label or aria-labelledby attribute.

(lint/a11y/noSvgWithoutTitle)